ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -396,6 +396,34 @@ EOF
   done
 }
 
+function inject_etcd_local_disk_for_openstack(){
```

Comment:
Can this be moved to a separate step? It would make PR approvals easier tbh

Reply:
I had a look but no, the `create manifests` and `create cluster` happen in the same step.
That's why other platforms also have this function in this step...

Reply:
Oh, I see, okay